osler/scripts/init_db.py

```diff
@@ -42,12 +42,6 @@
                             signs_charts=can_sign, staff_view=part_staff)
     p.save()
 
```

Comment:
I'm pretty sure we can get rid of this file entirely. It's a relic from pre-Docker